config-example.sh

```diff
@@ -57,8 +57,9 @@ export DEFAULTFOURDRIVESWRAIDLEVEL="6"
 export DEFAULTLVM="0"
 export DEFAULTLOADER="grub"
 export DEFAULTGOVERNOR="ondemand"
-
+export FORCE_GPT=1
 export V6ONLY="0"
+export DWC_ENABLE=1
```

Comment:
same here with `declare -x -i`

Reply:
great advice, I have updated the PR accordingly. thanks!